sparkplug/config/consumer.py

```diff
@@ -87,20 +88,37 @@ def __init__(self, name, use, queue, **kwargs):
         DependencyConfigurer.__init__(self)
         self.entry_point = self.parse_use('sparkplug.consumers', use)
         self.queue = queue
+
         self.consumer_params = kwargs
-        
+
+        # timerreporters are optional, but calling this is not optional:
+        self._init_reporters()
+
         self.depends_on(queue)
-    
+
+
+    def _init_reporters( self ):
+        self.time_reporters = []
+        if 'time_reporters' in self.consumer_params:
+            self.time_reporters = [ x.strip() for x in self.consumer_params['time_reporters'].split(',') ]
+            for name in self.time_reporters:
+                self.depends_on(name)
+            del self.consumer_params['time_reporters']
```

Comment:
Do we clean this up just to make sure that it doesn't get used again?

Reply:
I added a comment to clarify.  This is just to keep the parameter data structure consistent with what downstream entities expect.